src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabView.cs

```diff
@@ -608,6 +595,18 @@ ItemsPresenter GetItemsPresenter(ListView listView)
 						scrollViewer.UnregisterPropertyChangedCallback(ScrollViewer.ScrollableWidthProperty, scrollViewerScrollableWidthToken);
 					});
 				}
+
+				if (ReadLocalValue(SelectedItemProperty) != DependencyProperty.UnsetValue)
```

Comment:
As this is a difference compared to original WinUI sources, we need to understand why the code needs to be moved, and if no way around it, it needs to be documented in the code with `// Uno specific` comment and details. If the problem only occurs for iOS, it may not be needed to move this on the other targets

Reply:
@MartinZikmund  The change was not required; I reverted the change to keep the code in Sync with WinUI.